assets/section-main-product.css

```diff
@@ -1244,3 +1244,80 @@ a.product__text {
   margin-left: 1.2rem;
   flex-shrink: 0;
 }
+
+/* Icon with text */
+.icon-with-text {
+  --icon-size: calc(var(--font-heading-scale) * 3rem);
+  --icon-spacing: calc(var(--font-heading-scale) * 1rem);
+}
+
+.icon-with-text--horizontal {
+  display: flex;
+  justify-content: center;
+  column-gap: 3rem;
+  flex-direction: row;
+  padding-top: 2rem;
+}
+
+.icon-with-text--vertical {
+  --icon-size: calc(var(--font-heading-scale) * 2rem);
+  padding-top: 1.5rem;
+}
+
+.icon-with-text .icon {
+  fill: rgb(var(--color-foreground));
+  height: var(--icon-size);
+  width: var(--icon-size);
+}
+
+.icon-with-text--horizontal .icon {
+  margin-bottom: var(--icon-spacing);
+}
```

Comment:
Looks like we can consolidate a couple other selectors into this rule.
`.icon-with-text--horizontal img` below on line 1289 and `.icon-with-text--vertical .icon-with-text__item` on line 1321
```
.icon-with-text--horizontal .icon,
.icon-with-text--horizontal img,
.icon-with-text--vertical .icon-with-text__item {
  margin-bottom: var(--icon-spacing);
}
```

Reply:
Yep I may have been crazy, disregard 🙂